### ISSUE TITLE
positive attribute of `upward_air_velocity`

### ISSUE THREAD

**Issue by larsbuntemeyer** [PCMDI/cmip6-cmor-tables#421]
Basically a duplication of

* https://github.com/WCRP-CORDEX/data-request-table/issues/47

`upward_air_velocity` should probably have the positive = up attribute. This is not the case in the [corresponding cmip6 cmor table](https://github.com/PCMDI/cmip6-cmor-tables/blob/6a30acaea9b63fdecbe08894d7e50ef36879a993/Tables/CMIP6_AERmon.json#L2069-L2085) but i guess from the logic it should have the positive attribute set...
(all other standard names containg one of ["incoming", "outgoing", "downward", "upward", "downwelling", "upwelling", "sinking"] are directional variables with a positive attribute set either up or down except for upward_air_velocity...)

**Comment by taylor13** [comment]
In the CMIP data request tables, a "positive" attribute that can be attached to a variable (like a flux of energy) has no grounding in CF.  The "positive" attribute in CF applies to coordinate variables (and their bounds) only.  The CMIP-defined "positive" attribute, although not a part of CF, does not make CMIP files non-compliant with CF because CF permits users to define "extra" metadata, which CF knows nothing about.

The "positive" attribute attached to some regular CMIP variables was introduced as an aid to modeling groups attempting to write model output conforming to the CMIP requirements.  For example, the CMOR code used by many modeling groups could determine which direction a flux was considered to be positive (from the "positive" attribute specified) without "reading" the standard_name.  The data writer could specify which direction was assumed to be positive for their own flux variable (through an argument passed to CMOR), and then CMOR could check to see whether the sign needed to be reversed before writing the variable to a  CMIP-compliant file.  

Bottom line is that for CMIP, yes, positive=up could be defined for this variable, but it is irrelevant to the CF conventions.

**Comment by larsbuntemeyer** [comment]
Yes, agreed! Thanks for clarifications!
